include/hal/library/responder/setcertlib.h

```diff
@@ -22,8 +22,9 @@
 extern bool libspdm_is_in_trusted_environment();
 
 /**
- * Stores a certificate chain in non-volatile memory.
- *
+ * Stores or erase a certificate chain in non-volatile memory.
+ * If the cert_chain is NULL and cert_chain_size, base_hash_algo and base_asym_algo are 0,
```

Comment:
I think at least we need check `cert_chain` and `cert_chain_size`.

This is a good practice to prevent some user mistake, e.g. allocate_copy cert_chain but without check the return NULL value.

I agree to ignore base_hash_algo and base_asym_algo check.

Reply:
Thanks. I have changed the code and comment for the API.